datadog-logs-oci-orm/modules/containerregistry/login.tf

```diff
@@ -0,0 +1,91 @@
+resource "null_resource" "get_username" {
```

Comment:
we can't use the terraform [resource for this](https://registry.terraform.io/providers/oracle/oci/latest/docs/data-sources/identity_user) ?

Reply:
I think we should be able to. I will experiment with that, and change it if that works.

Reply:
updated